core/domain/question_services.py

```diff
@@ -177,16 +194,16 @@ def get_question_by_id(question_id, strict=True):
 
 
 def get_question_skill_links_of_skill(skill_id):
-    """Returns a list of QuestionSkillLinkModels of
+    """Returns a list of QuestionSkillLinks of
```

Comment:
Docstring doesn't match what the implementation is doing, I think?

But you're right, this ought to be returning domain objects, not datastore models. Could you please update the implementation to do this (and ensure the unit tests reflect the new contract)?

Reply:
fixed